File: 02/2.py
a,b,total= 0,1,0
for i in range (0,4000000):
	current = a + b
	b = a
	a = current 
	if current >= 4000000:
		break
	if current < 4000000 and  current % 2 == 0:
		total = total + current
print("Answer:",total)

File: 02/2.java
class main 
{
    public static void main(String[] args) 
	{
        int total = 0 , a = 0 , b = 1;
		while (a <= 4000000)
		{
			int tmp = a + b;
			b = a;
			a = tmp;
			if (a % 2 == 0)
				total += a;
		}
        System.out.println("Answer: "+total);
    }
}
